docs/source/howto/workflows.rst

```diff
@@ -20,6 +20,159 @@ Running a predefined workflow
 
 `#3992`_
 
+The first step to running a predefined workflow is loading the work function or work chain class that defines the workflow you want to run.
+This can of course be achieved by importing the function or class from its respective module:
+
+.. code-block:: python
+
+    from aiida.workflows.plugins.arithmetic.addmultiply import add_and_multiply              # Import the addmultiply work function
+    from aiida.workflows.plugins.arithmetic.multiplyadd import MultiplyAndAddWorkChain       # Import the MultiplyAndAddWorkChain WorkChain
```

Comment:
As we discussed earlier, the motivation for doing the addition after the multiplication in the `MultiplyAndAddWorkChain` was twofold:

* The tutorial first shows the calculation function and then the `ArithmeticAddCalculation` calculation job, using the output of the calculation function. In order to be consistent when explaining the work chain, I used the same order. Of course, we can also write a `ArithmeticMultiplyCalculation` calculation job to solve this.
* `AddAndMultiplyWorkChain` is already used in other parts of the documentation (e.g. [here](https://aiida-core.readthedocs.io/en/latest/working/workflows.html#work-chains)), and is defined as a work chain that only calls calculation functions. 

Perhaps it would be good if I opened an issue to discuss the *arithmetic* processes, where we can settle on which set to use and that I can refer to when I do the PR into `develop`?

On the second point: agreed!  I'll refactor the code and leave mirrors with a deprecation warning.

Reply:
As discussed, `MultiplyAddWorkChain` and `add_multiply` for the workfunction with `multipy_add` and `add_multiply` as the two entry points names is good for me :+1: